src/test/java/fi/otavanopisto/kuntaapi/test/server/unit/discover/EntityUpdaterTest.java

```diff
@@ -0,0 +1,47 @@
+package fi.otavanopisto.kuntaapi.test.server.unit.discover;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+import java.util.Iterator;
+
+import javax.ejb.Singleton;
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Instance;
+import javax.inject.Inject;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.deltaspike.testcontrol.api.junit.CdiTestRunner;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import fi.otavanopisto.kuntaapi.server.discover.EntityUpdater;
+
+@RunWith (CdiTestRunner.class)
+public class EntityUpdaterTest {
+  
+  @Inject
+  private Instance<EntityUpdater> entityUpdaters;
+  
+  @Test
+  @SuppressWarnings("unchecked")
+  public void testBeanAnnotations() throws IOException, ClassNotFoundException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Refactor this method to throw at most one checked exception instead of: java.io.IOException, java.lang.ClassNotFoundException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1160)